Example/src/screens/Rows.tsx

```diff
@@ -88,19 +119,20 @@ const Rows = () => {
                     maxWidth: 400,
                 }}
             >
-                <SectionList
+                <UICollapsableSectionList
                     contentContainerStyle={{ paddingHorizontal: 16 }}
                     sections={sections}
                     renderItem={renderUIListItem}
-                    renderSectionHeader={({ section: { title } }) => (
-                        <UILabel role={UILabelRoles.HeadlineHead}>{title}</UILabel>
-                    )}
+                    renderSectionHeader={renderSectionHeader}
+                    stickySectionHeadersEnabled={false}
+                    keyExtractor={item => item.key}
+                    windowSize={5}
                 />
-                <UIBoxButton
+                {/* <UIBoxButton
```

Comment:
Why commenting it out? I thought it's good for testing

Reply:
Just was annoying during development :)